pkgs/applications/science/biology/star/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "star";
+  version = "2.5.3a";
+
+  src = fetchFromGitHub {
+    repo = "STAR";
+    owner = "alexdobin";
+    rev = version;
+    sha256 = "1fd9xl7i1zxgsxn2qf6gz8s42g2djm29qmp6qb35d8nnxh8ns54x";
+  };
+
+  buildInputs = [ zlib ];
+
+  buildPhase = ''
+    cd source
+    sed 's/\/bin\/rm/rm/g' Makefile > Makefile2
```

Comment:
This should go into `postPatch`.

Reply:
Use another separator to avoid escaping slashes. Also, use inline sed processing to avoid extra `mv`.
```sh
sed 's:/bin/rm:/rm/g' -i Makefile
```